step-4/client/app/components/navigation/navigation.component.js

```diff
@@ -39,6 +41,11 @@ export const NavigationComponent = {
       function mapStateToThis(state) {
         return {
           allTypes: TypesSelectors.getAllTypes(state),
+
```

Comment:
Now there are. Not for the types but for the Todos. I needed to be aware of the Todos when deleting the type on the navigation menu.